packages/cli/src/constructs/private-location.ts

```diff
@@ -0,0 +1,80 @@
+import { Construct } from './construct'
+import { Session } from './project'
+import { ValidationError } from './validator-error'
+
+export interface PrivateLocationProps {
```

Comment:
Don't forget to add the `icon` field as well. We can add the string literals later on of course